packages/atlaskit/src/components/fields/Date.js

```diff
@@ -38,6 +39,7 @@ class AtlaskitDate extends React.Component<Field> {
           value={value}
           isDisabled={disabled}
           onFocus={() => onFieldFocus(id)}
+          autoFocus={autoFocus}
```

Comment:
I think it would be better to call this 'autofocus' to be more consistent with naming or similar HTML attributes to prevent confusion (see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus)

Reply:
just to clarify, do you mean smth like `autofocus={autoFocus}`?

Reply:
done